### PR TITLE
tags and justificatif/commentary are displaying correctly

### PR DESCRIPTION
**Pull request by RMayeux**

### CODE REVIEW

**Review by clemcx — APPROVED** [review]
ok approved

File: templates/cvAdvisor.html.twig [hunk 1]
```diff
@@ -23,32 +23,54 @@
                     </div>
                     <div class="blocInfo" id="competences">
                         <h2>Compétences clés</h2>
-                        <p>Marketing / Communication & réseaux sociaux, Stratégie / Business Developpement / Vision stratégique</p>
+                        <div class="tags">
+                            {% for tag in advisor.tags %}
+                                {% if tag.category.id == 2 %}
```
Comment (benj-san) [inline templates/cvAdvisor.html.twig]: Rather use a name than an id (imagine you have to change the database)

**Review by benj-san — COMMENTED** [review]
Please fix the comment before approval

File: templates/cvAdvisor.html.twig (hunk 1 above)
Comment (RMayeux) [reply templates/cvAdvisor.html.twig]: Fixed

**Review by benj-san — APPROVED** [review]

**Pull request merged**